pnpm-workspace.yaml

```diff
@@ -44,3 +44,5 @@ onlyBuiltDependencies:
   - "@hyperledger/anoncreds-nodejs"
   - "@hyperledger/indy-vdr-nodejs"
   - "@openwallet-foundation/askar-nodejs"
+  - "zstd-napi"
+  - "cbor-extract"
```

Comment:
what are these libraries used for?

Reply:
On Hedera, AnonCreds resources are stored according to [HCS-1 standard](https://hashgraphonline.com/docs/standards/hcs-1) that in our case makes use of zstd compression.
`zstd-napi` is a dependency for Node JS environment and also a dev dependency of Hiero DID SDK.

Just in case, for React Native environment we're using `react-native-zstd` package instead.
The SDK resolves either `zstd-napi` or `react-native-zstd` based on environment (using both as optional/dynamic dependencies).

Regarding `cbor-extract`, it's a transitive dependency of Hiero DID SDK JS. The SDK supports CBOR encoding for DID resolver result ("application/did+cbor") and uses [cbor-x](https://github.com/kriszyp/cbor-x) for encoding/decoding.